extensions/shared/register-jetpack-block.js

```diff
@@ -1,16 +1,29 @@
 /**
  * External dependencies
  */
+import { endsWith } from 'lodash';
 import { registerBlockType } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import extensionList from '../index.json';
 import getJetpackExtensionAvailability from './get-jetpack-extension-availability';
+import wrapPaidBlock from './wrap-paid-block';
 
 const betaExtensions = extensionList.beta || [];
 
+function requiresPlan( unavailableReason ) {
+	if ( endsWith( unavailableReason, '_plan_required' ) ) {
```

Comment:
It's polyfilled, try `'abc'.endsWith('c')` is `true` from block editor in ie11.

```suggestion
	if ( unavailableReason.endsWith( '_plan_required' ) ) {
```

That said, externalized lodash is all or nothing and it's there in the block-editor as well, you can run `lodash.endsWith( 'abc', 'c' )` in the ie11 console, so this isn't too important.

Reply:
Obsolete per 87d15d8